test/test_images/latencymako/dev.config

```diff
@@ -33,11 +33,19 @@ metric_info_list: {
   value_key: "pe"
   label: "publish-errors"
 }
+metric_info_list: {
+  value_key: "st"
+  label: "send-throughput"
```

Comment:
Can this be called `publish-throughput` to match the others? Or rename the others to `send-*` if you want.

Reply:
it's not the same imo :) with send i mean that i measure it in the moment when i "expect" to send it

Reply:
Ok, sure. If there's a subtlety there, can you add a comment explaining the difference between send and publish? (and it can go in the followup PR for #1922)

Reply:
next pr i'll put it :)